base/array.jl

```diff
@@ -127,15 +136,45 @@ asize_from(a::Array, n) = n > ndims(a) ? () : (arraysize(a,n), asize_from(a, n+1
 """
     Base.isbitsunion(::Type{T})
 
-Return whether a type is an "is-bits" Union type, meaning each type included in a Union is `isbitstype`.
+Return whether a type is an "is-bits" Union type, meaning each type included in a Union is [`isbitstype`](@ref).
+
+# Examples
+```jldoctest
+julia> ut = Union{Float64, UInt8}
```

Comment:
`;`

Reply:
Also, is there a need to create the variable at all here?

Reply:
No, probably cleaner and more concise without it.